lib/src/main/java/co/infinum/goldfinger/CryptoObjectInitRunnable.java

```diff
@@ -29,10 +33,18 @@ public void run() {
             case ENCRYPTION:
                 cryptoObject = cryptoFactory.createEncryptionCryptoObject(keyName);
                 break;
+            default:
+                cryptoObject = null;
```

Comment:
This means that we need to handle null values afterwards. Is AsyncCryptoFactory.Callback used only here? If it is, can we add to javadocs when can we expect the null value?

If it's also used somewhere else, maybe we could enhance the Callback with a onError() callback, which would receive some error code/message/description.

Reply:
We handled them earlier also. If cryptoObject is null it means that we failed to initialize it. Javadoc already exists in this interface

Reply:
👍